app/models/zip_part.rb

```diff
@@ -16,7 +16,7 @@ class ZipPart < ApplicationRecord
     'replicated_checksum_mismatch' => 3
   }
 
-  validates :zipped_moab_version, :create_info, presence: true
+  validates :create_info, presence: true
```

Comment:
Why are we dropping this validation?

Reply:
Rails/RedundantPresenceValidationOnBelongsTo